### PR TITLE
Convert RulesetToEditorConfigConverter to use top-level statements

### PR DESCRIPTION
**Pull request by sharwell**

### CODE REVIEW

File: src/Tools/RulesetToEditorconfigConverter/Source/Program.cs [hunk 1]
```diff
@@ -1,42 +1,37 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+#pragma warning disable IDE0051 // Remove unused private members
```
Comment (mavasani) [inline src/Tools/RulesetToEditorconfigConverter/Source/Program.cs:3]: Isn't this bug fixed yet?

File: src/Tools/RulesetToEditorconfigConverter/Source/Program.cs [hunk 2]
```diff
@@ -1,42 +1,37 @@
...
+#pragma warning disable CA1812 // Avoid uninstantiated internal classes
```
Comment (mavasani) [inline src/Tools/RulesetToEditorconfigConverter/Source/Program.cs:4]: Can you file an issue for this CA1812 false positive?

File: src/Tools/RulesetToEditorconfigConverter/Source/Program.cs (hunk 1 above)
Comment (sharwell) [reply src/Tools/RulesetToEditorconfigConverter/Source/Program.cs:3]: It was still getting reported in 16.7 Preview 3. I'm assuming that dotnet/roslyn#44848 will allow this to be cleaned up when the IDE is updated.

Comment (sharwell) [reply src/Tools/RulesetToEditorconfigConverter/Source/Program.cs:3]: Looks like https://github.com/dotnet/roslyn/issues/45496 might not be fixed yet.

**Comment by mavasani** [comment]
@sharwell Can you resolve conflicts and fix build errors?

**Comment by sharwell** [comment]
@mavasani Last time around build errors appeared to be caused by CI using an older compiler. Has that been fixed?

**Comment by mavasani** [comment]
> Last time around build errors appeared to be caused by CI using an older compiler. Has that been fixed?

We did move to a newer compiler recently:
https://github.com/dotnet/roslyn-analyzers/blob/4da462a69dec7e5ce877dcb8fecb84f94ffe7bee/eng/Versions.props#L27

**Comment by mavasani** [comment]
@sharwell - we already converted another project to use top level statements in this repo. So this PR should be unblocked.

**Comment by sharwell** [comment]
@mavasani this is now updated

**Pull request merged**